components/articles/single-article.jsx

```diff
@@ -19,20 +20,21 @@ useEffect(()=>{
 const date = convertToDates(singleArticle.created_at)
 
 if (isLoading) {
-    return <h2>Fetching the news for you</h2>
+    return <h2>Fetching...</h2>
 }
 else {
     return (
-    <section className="article-card">
+    <body className="single-article">
         <p>Article No.{singleArticle.article_id}</p>
         <h2>{singleArticle.title}</h2>
         <p>Topic: {singleArticle.topic}</p>
         <img className="articleImg" src={singleArticle.article_img_url}/>
         <p>Written by: {singleArticle.author}</p>
-        <p>Published on {date[0]}-{date[1]}-{date[2]}, {date[3]}:{date[4]}</p>
-        <p>{singleArticle.body}</p>
+        <p>Published on {date[0]}-{date[1]}-{date[2]}, {date[3] >= 0 && date[3] < 12 ? <>{date[3]}:{date[4]}AM</> : <>{date[3]}:{date[4]}PM</> }</p>
+        <section>{singleArticle.body}</section>
         <p>{singleArticle.comment_counts} comments</p>
```

Comment:
Having comment count at the bottom of your article is a really nice touch. I like that when I go to article 37 which has no comments, I can clearly see that there are no comments instead of wondering if the comments aren't loading. Really nice work!